Tasks/Common/azure-blobstorage-downloader/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "azure-blobstorage-downloader",
```

Comment:
Is it downloader? I believe it is used for both download/upload. Also update the description with appropriate name. azure-blobstorage-artifactProvider could be the potential name.